src/collapse/guess.rs

```diff
@@ -113,7 +114,7 @@ impl Collapse for Folder {
     #[cfg(test)]
     fn set_nstacks_per_job(&mut self, _: usize) {}
 
-    #[cfg(test)]
+    #[doc(hidden)]
```

Comment:
Some of these appear to be unrelated to the merge?

Reply:
Yes - See answer on main thread. If you want to respond, I guess it's best here; so I'll leave this unresolved.